### PR TITLE
refac: don't use external parameters that specifies container paths

### PR DESCRIPTION
**Pull request by JesperGomes**
## References

* #508

### CODE REVIEW

**Review by BjarkeMeier — APPROVED** [review]
Nice seeing you constantly improving the code standard

**Comment by BjarkeMeier** [comment]
Can you add an issue reference?

File: source/databricks/package/calculator_job.py [hunk 1]
```diff
@@ -12,21 +12,23 @@
...
-from pyspark.sql import SparkSession, DataFrame
-from pyspark.sql.functions import col
-from pyspark.sql.types import Row
 
+import configargparse
 from package import (
     calculate_balance_fixing_total_production,
+    db_logging,
+    debug,
+    infrastructure,
```
Comment (BjarkeMeier) [inline source/databricks/package/calculator_job.py:22]: Nice, lexically ordering these. Is it done by you or the tooling?

File: source/databricks/tests/integration/calculator/test_calculator_job.py [hunk 2]
```diff
@@ -39,25 +39,21 @@ def __init__(self, in_dict: dict):
...
-            "batch_id": executed_batch_id,
-            "batch_grid_areas": [805, 806],
-            "batch_snapshot_datetime": timestamp_factory("2022-09-02T21:59:00.000Z"),
-            "batch_period_start_datetime": timestamp_factory(
-                "2022-04-01T22:00:00.000Z"
-            ),
-            "batch_period_end_datetime": timestamp_factory("2022-09-01T22:00:00.000Z"),
-            "time_zone": "Europe/Copenhagen",
-        }
+    return CalculatorArgs(
```
Comment (BjarkeMeier) [inline source/databricks/tests/integration/calculator/test_calculator_job.py:45]: Nice, can we now remove the `DictObj` class?

**Review by BjarkeMeier — APPROVED** [review]
Looks like a nice improvement to our code base.

File: source/databricks/package/calculator_job.py (hunk 1 above)
Comment (JesperGomes) [reply source/databricks/package/calculator_job.py:22]: Tooling. But I currently trigger it manually: right-click ->Sort imports

File: source/databricks/tests/integration/calculator/test_calculator_job.py (hunk 2 above)
Comment (JesperGomes) [reply source/databricks/tests/integration/calculator/test_calculator_job.py:45]: It's still used in for parsing 'calculation-file-paths.yml'.

**Pull request merged**